src/TableOfContents/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, Drawer, Icon } from '@blueprintjs/core';
+import cn from 'classnames';
+import * as React from 'react';
+import { useIsMobile } from '../_hooks/useIsMobile';
+import { ScrollContainer } from '../ScrollContainer';
+import { IContentsNode } from './types';
+
+export interface ITableOfContents {
+  // Precomputed list of nodes instead of items. This is useful if you want to customize the ordering of the tree.
+  contents: IContentsNode[];
+  rowRenderer?: (item: IContentsNode, DefaultRow: React.FC<ITableOfContentsItem>) => React.ReactElement;
+
+  // Padding that will be used for (default: 10)
+  padding?: string;
+  className?: string;
+
+  // Title of project
+  title?: string;
+
+  // Controls for the drawer functionality on mobile
+  isOpen?: boolean;
+  onClose?: () => void;
+
+  // Mobile breakpoint, default (true) is 786px, false disables Drawer
+  enableDrawer?: boolean | number;
+}
+
+export const TableOfContents: React.FunctionComponent<ITableOfContents> = ({
+  contents: _contents,
+  rowRenderer,
+  className,
+  padding = '10',
+  title,
+  isOpen = false,
+  // tslint:disable-next-line: no-empty
+  onClose = () => {},
+  enableDrawer = true,
+}) => {
+  const contents = _contents;
+
+  // TODO (CL): Should we store expanded state in local storage?
+  const [expanded, setExpanded] = React.useState({});
+
+  const isMobile = useIsMobile(enableDrawer);
+
+  const comp = (
+    <div className={cn('TableOfContents bg-gray-1 dark:bg-transparent flex justify-end h-full border-r', className)}>
```

Comment:
Looks like we're not adding a border-r in Elements. Do you remember why we added one here? If the consumer needs a border, they can just pass it in via the className prop?

https://github.com/stoplightio/elements/blob/master/src/components/TableOfContents/index.tsx#L79

```suggestion
    <div className={cn('TableOfContents bg-gray-1 dark:bg-transparent flex justify-end h-full', className)}>
```